### DISCUSSION TITLE
视频跟读，识别出来的单词有偏差，是否可以手动修改某个单词或者某个断句呢

### DISCUSSION THREAD

**Discussion by zaasasa** [ZuodaoTech/everyone-can-use-english#593] [Q&A]
视频跟读，识别出来的单词有偏差，是否可以手动修改某个单词或者某个断句呢

**Comment by an-lee** [comment]
[v0.2.4](https://github.com/xiaolai/everyone-can-use-english/releases/tag/v0.2.4) 中已支持修改语音文本
